book/visual-effects.md

```diff
@@ -1180,8 +1179,8 @@ commands to a surface, and *then* apply transparency to it when
 blending into the parent.
 
 ::: {.further}
-I highly recommend [this blog post](https://ciechanow.ski/alpha-compositing/),
-which gives a really nice visual overview of many of the same concepts explored in
+I highly recommend a [blog post by Bartosz Ciechanowski](https://ciechanow.ski/alpha-compositing/),
+that gives a really nice visual overview of many of the concepts explored in
```

Comment:
"this" really doesn't work in the book. On the web it's blue so it's obvious what it's referring to.